partiql-lang/src/main/kotlin/org/partiql/lang/eval/visitors/StaticTypeInferenceVisitorTransform.kt

```diff
@@ -1310,15 +1310,15 @@ internal class StaticTypeInferenceVisitorTransform(
             }
 
             val hasDuplicateKeys = structFields
-                .groupingBy { it.first }
+                .groupingBy { it.key }
                 .eachCount()
                 .any { it.value > 1 }
 
             if (hasDuplicateKeys) {
                 TODO("Duplicate keys in struct is not yet handled")
             }
```

Comment:
Are we planning on fixing the staticTypeInferencer in the future?

Reply:
I opted to defer as I don't know the future of the StaticTypeInferencer.